src/scss/_retailers.scss

```diff
@@ -0,0 +1,134 @@
+//---------------Break Points---------------//
+//50em (800px)
+//61.25em (980px)
+//72em (1152px)
+//75em (1200px)
+
+.retailers {
+	background: url("../images/tan-wood.jpg") no-repeat center;
+	background-size: cover;
+	text-align: center;
+	padding: 4em 0; //64px 0
+}
+
+.retailers__heading {
+	width: 10em; //160px
+	font-family: $font-egyptian;
+	font-size: 1em; //16px
+	color: $color-blue;
+	border-top: 2px solid $color-blue;
```

Comment:
I like that you've labeled your color variables as "$color..." makes it really easy to know what it is